src/cets_call.erl

```diff
@@ -110,3 +126,24 @@ where(Name) when is_atom(Name) -> whereis(Name);
 where({global, Name}) -> global:whereis_name(Name);
 where({local, Name}) -> whereis(Name);
 where({via, Module, Name}) -> Module:whereis_name(Name).
+
+%% Sends all requests to a single node in the cluster
+-spec send_leader_op(server_ref(), op()) -> term().
+send_leader_op(Server, Op) ->
+    Leader = cets:get_leader(Server),
+    Res = cets_call:sync_operation(Leader, {leader_op, Op}),
+    case Res of
+        {error, wrong_leader} ->
+            ?LOG_WARNING(#{what => wrong_leader, server => Server, operation => Op}),
+            timer:sleep(10),
```

Comment:
it is 10 milliseconds delay. But I could retry without delay, it will just make our local server less happy during the join operation (and extra load for the remote server too). Alternatively, I can wait for something like "join finished" event.

Will apply retry without delay, add a smarter solution in a separate PR.

Reply:
The only issue is that a bug could cause it to spin forever, at least in theory.